lib/service/service.go

```diff
@@ -3244,6 +3244,17 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 	return nil
 }
 
+// kubeDialAddr returns Proxy Kube service address used for dialing local kube service
+// by remote trusted cluster.
+// If the proxy is running with V2 configuration the WebPort is returned
+// where connections are forwarded to kube service by ALPN SNI router.
+func kubeDialAddr(config *Config) utils.NetAddr {
+	if config.Version == defaults.TeleportConfigVersionV2 {
```

Comment:
Hm, shouldn't this check for listener multiplex mode rather than config version?

Reply:
updated.